ush/parsing_namelists_WW3.sh

```diff
@@ -114,213 +104,60 @@ WW3_namelists(){
     exit 12 
   fi
 
-
-
-if [ $waveMULTIGRID = ".true." ]; then
```

Comment:
I hope the `else` part of this has been kept up-to-date.  See comment below.

This will be a major simplification